rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2705,7 +2705,8 @@ public Coin getEstimatedFeesForNextPegOutEvent() throws IOException {
 
         int pegoutRequestsCount = getQueuedPegoutsCount();
 
-        if (!activations.isActive(RSKIP271) || pegoutRequestsCount == 0) {
+        if (!activations.isActive(ConsensusRule.RSKIP385) &&
+                (!activations.isActive(RSKIP271) || pegoutRequestsCount == 0)) {
```

Comment:
There's No Need Checking If The Count Is Zero, Because Now We Want To Ignore That And Calculate The Cost of a Pegout With 2 Outputs (one for the pegout the user wants to make and another for the change)